changelog.txt

```diff
@@ -0,0 +1,3 @@
+changed - `firebase` is no longer a direct dependency.
```

Comment:
Suggestion: "`firebase` is no longer a peer dependency. You can use either the `firebase` or `firebase-admin` module with Firebase Queue."